condensedinlinepanel/edit_handlers.py

```diff
@@ -167,24 +167,26 @@ def __init__(self, relation_name, panels=None, heading='', label='', help_text='
         self.card_header_from_field = card_header_from_field
         self.card_header_from_js = card_header_from_js
         self.card_header_from_js_safe = card_header_from_js_safe
+        self.new_card_header_text = new_card_header_text
 
     def bind_to_model(self, model):
         if django.VERSION >= (1, 9):
             related = getattr(model, self.relation_name).rel
         else:
             related = getattr(model, self.relation_name).related
 
-        related_name = camelcase_to_underscore(
-            related.related_model.__name__).replace("_", " ").title()
+        heading = self.heading or _(related.related_model._meta.verbose_name_plural.capitalize())
+        new_card_header = self.new_card_header_text or _("New %s" % related.related_model._meta.verbose_name)
+        label = self.label or _('Add %s' % related.related_model._meta.verbose_name.capitalize())
```

Comment:
Will drop the capitalising.

Also will redo the translations. I actually haven't played with translations much and didn't actually realise that in your examples you were calling `.format` after the `_` call. I've just had a quick read through the django docs and realised that with their example:
```
output = _('Today is %(month)s %(day)s.') % {'month': m, 'day': d}
```

I'll update with a new commit.

Although because I find it less verbose and easier to read, I tend to stick with old style string formatting unless I explicitly need new style features, so apologies in advance for that. They can take it away from me when they deprecate it (which probably won't happen). :P

Reply:
Yes, I must admit I still find myself using both in different places. I have a mild preference for format() recently though. Happy with this 😀